sgxs-loaders/src/sgx_enclave_common/mod.rs

```diff
@@ -244,13 +246,26 @@ impl EnclaveLoad for InnerLibrary {
             ) {
                 return Err(Error::Init(error.into()));
             }
-
+            #[cfg(unix)]
             libc::mprotect(
                 mapping.base as _,
                 mapping.size as _,
                 libc::PROT_READ | libc::PROT_WRITE | libc::PROT_EXEC,
             );
 
+            #[cfg(windows)]
+                {
+                    let mut prev_protection : winapi::shared::minwindef::DWORD = 0;
+                    if winapi::um::memoryapi::VirtualProtect(
```

Comment:
Did you try without this and it failed?

Reply:
I don't think I had tried it before. I remember it was mostly done to ensure the same steps happen for both OS

I tried it now.
It works correctly without the VirtualProtect function.

Reply:
Should I remove it or keep it as it is?

Reply:
Remove. I don't remember why the mprotect was necessary on Linux, but it is.